olive/evaluator/metric.py

```diff
@@ -12,16 +12,28 @@
 from olive.common.config_utils import ConfigBase, ConfigDictBase, validate_config
 from olive.data.config import DataConfig
 from olive.evaluator.accuracy import AccuracyBase
-from olive.evaluator.metric_config import LatencyMetricConfig, MetricGoal, get_user_config_class
+from olive.evaluator.metric_config import LatencyMetricConfig, MetricGoal, ThroughputMetricConfig, get_user_config_class
 
 logger = logging.getLogger(__name__)
 
 
 class MetricType(str, Enum):
     ACCURACY = "accuracy"
     LATENCY = "latency"
+    THROUGHPUT = "throughput"
```

Comment:
I was considering that before. But it will be hard to compare the metrics, as for latency, we expect lower is better, but for throughput, higher is better.
Also it will be more clear use separated way even we need to compute latency twice.

Reply:
Oh yeah, I forgot that the direction if opposite. Sounds good.